eng/Versions.props

```diff
@@ -119,8 +119,9 @@
     <MonoAddinsPackageVersion>1.3.8</MonoAddinsPackageVersion>
     <MonoDevelopSdkPackageVersion>1.0.15</MonoDevelopSdkPackageVersion>
     <MoqPackageVersion>4.10.0</MoqPackageVersion>
-    <!-- STOP!!! We need to reference the version of JSON that our HOSTS supprt. -->
+    <!-- STOP!!! We need to reference the version of JSON that our HOSTS support. -->
     <NewtonsoftJsonPackageVersion>12.0.2</NewtonsoftJsonPackageVersion>
+    <StreamJsonRpcPackageVersion>2.7.62-alpha</StreamJsonRpcPackageVersion>
```

Comment:
Needed for activity tracing, version matches dotnet/roslyn.

Reply:
Looks like @ryanbrandenburg's PR will make this obsolete. 

https://github.com/dotnet/aspnetcore-tooling/pull/3110/files#diff-1ea18ff65faa2ae6fed570b83747086d0317f5e4bc325064f6c14319a9c4ff67R127

If that PR is merged first, will just remove this line.

Reply:
https://github.com/dotnet/aspnetcore-tooling/pull/3116/commits/4f4ad5b30abdfd3c160faa44f0f37550d1d82f59